test/run.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# shellcheck disable=SC2086
+
+ROOTDIR=$(git rev-parse --show-toplevel)
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+RF_VENV="${ROOTDIR}/_output/robotenv"
+RF_BINARY="${RF_VENV}/bin/robot"
+RF_VARIABLES="${SCRIPTDIR}/variables.yaml"
+
+DRYRUN=false
+OUTDIR="${ROOTDIR}/_output/e2e-$(date +%Y%m%d-%H%M%S)"
+
+
+function usage {
+    local -r script_name=$(basename "$0")
+    cat - <<EOF
+${script_name} [-h host] [-n] [-o output_dir] [test suite files]
+
+Options:
+
+  -h       Print this help text.
+
+  -n       Dry-run, do not run the tests.
+
+  -o DIR   The output directory. (${OUTDIR})
+
+EOF
+}
+
+while getopts "h:no:" opt; do
+    case ${opt} in
+        h)
+            usage
+            exit 0
+            ;;
+        n)
+            DRYRUN=true
+            ;;
+        o)
+            OUTDIR=${OPTARG}
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ ! -f "${RF_VARIABLES}" ]; then
+    echo "Please create a variables file at ${RF_VARIABLES}" 1>&2
+    echo "See ${RF_VARIABLES}.example for the expected content." 1>&2
+    exit 1
+fi
```

Comment:
We renamed the default `yaml`, so one needs to make changes to the project for running scripts. 
Can we avoid that by having the defaults in place for running in `devenv`?

Reply:
I added some documentation explaining how to manage the setup